File: artemis-entity-tracker-gui/src/main/java/net/namekdev/entity_tracker/ui/listener/ChangingSystemEnabledStateListener.java
package net.namekdev.entity_tracker.ui.listener;

import net.namekdev.entity_tracker.ui.model.BaseSystemTableModel;

public interface ChangingSystemEnabledStateListener extends java.util.EventListener {
	void onChangingSystemEnabledState(BaseSystemTableModel model, int systemIndex, String systemName, boolean enabled);
}

File: artemis-entity-tracker-gui/src/main/java/net/namekdev/entity_tracker/ui/listener/ChangingSystemEnabledStateListener.kt
package net.namekdev.entity_tracker.ui.listener

import net.namekdev.entity_tracker.ui.model.BaseSystemTableModel

interface ChangingSystemEnabledStateListener : java.util.EventListener {
    fun onChangingSystemEnabledState(model: BaseSystemTableModel, systemIndex: Int, systemName: String, enabled: Boolean)
}
